cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/StatementHandler.kt

```diff
@@ -76,14 +77,52 @@ class StatementHandler(frontend: PythonLanguageFrontend) :
     private fun handleImport(node: Python.ASTImport): Statement {
         val declStmt = newDeclarationStatement(rawNode = node)
         for (imp in node.names) {
-            val v =
-                if (imp.asname != null) {
-                    newVariableDeclaration(imp.asname, rawNode = imp) // TODO refers to original????
+            val alias = imp.asname
+            val decl =
+                if (alias != null) {
+                    newImportDeclaration(
+                        parseName(imp.name),
+                        false,
+                        parseName(alias),
+                        rawNode = imp
+                    )
+                } else {
+                    newImportDeclaration(parseName(imp.name), false, rawNode = imp)
+                }
+            frontend.scopeManager.addDeclaration(decl)
+            declStmt.addToPropertyEdgeDeclaration(decl)
+        }
+        return declStmt
+    }
+
+    private fun handleImportFrom(node: Python.ASTImportFrom): Statement {
+        val declStmt = newDeclarationStatement(rawNode = node)
+        val module =
```

Comment:
Maybe I don't get Kotlin, but is this really happening here? Also: `from ..foo.bar import...`

Reply:
We need to check the import "size" or however it is called here.